src/components/Sticky/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { PureComponent } from 'react'
```

Comment:
One of the main components of the original issue / feature request was being able to set an offset from the top.  We currently have a fixed header on body scroll, so we'd have to write additional logic on top of this sticky component to figure out the offset.  The original experiment's implementation looked at scroll / offsets which you can see here:

https://github.com/jstnjns/masterclass/blob/react-homepage/app/javascript/components/Fixed/index.js#L34

I think this is something we should support, but @jstnjns can weigh in on whether that's needed for this particular sticky component.

Reply:
@stephenp, so @jstnjns and I discussed the 'showAfter' property this morning and he feels this particular behavior should be encapsulated as a separate component.  I can create another PR for that component.  I can add another property to this PR to handle the offset

Reply:
yea.  sry @stephenp didn't update you on this.  realized that it really was separate functionality, and that we shouldn't conflate the utility of this component with the utility of what should be another component

Reply:
@jstnjns @stephenp would we want to call such a component 'ShowAfter' ?

Reply:
or maybe 'RevealOnScroll' ?

Reply:
https://github.com/yankaindustries/mc-components/issues/578

For ease, I think it's fine to start with a pixel value.  If we REALLY need to pull in a ref or something, we can improve that later.